aira/src/aiq_aira/utils.py

```diff
@@ -27,20 +54,23 @@ def update_system_prompt(system_prompt: str, llm: ChatOpenAI):
 
     return system_prompt
 
+
 def get_domain(url: str):
     """
     Extract the domain from a URL.
     """
     domain = url.split("/")[2]
     return domain.replace("www.", "") if domain.startswith("www.") else domain
 
+
 async def dummy():
     """
     A do-nothing async function for placeholders.
     """
     return None
 
-def format_sources(sources: str) -> str:
+
+def format_sources(sources: str, source_num_start: int | None = None) -> str:
```

Comment:
the function definition does not use source_num_start, we can remove this parameter as well